src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -412,7 +414,7 @@ private void setupGlState(float[] orthoMatrix) {
 		// Set up our orthographic projection matrix and load it into the legacy matrix stack
 		RenderSystem.matrixMode(GL11.GL_PROJECTION);
 		RenderSystem.pushMatrix();
-		GL11.glLoadMatrixf(orthoMatrix);
+		IrisRenderSystem.loadMatrixf(orthoMatrix);
```

Comment:
I think it would make sense to move this whole code block to IrisRenderSystem and call it "setProjectionMatrix" - then we could have the other code block be "restoreProjectionMatrix"

Reply:
Done.